packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -33,16 +33,16 @@ export abstract class Metric<T> implements types.Metric<T> {
   }
 
   /**
-   * Returns a Handle associated with specified label values.
+   * Returns a Handle associated with specified LabelSet.
    * It is recommended to keep a reference to the Handle instead of always
    * calling this method for each operation.
-   * @param labelValues the list of label values.
+   * @param labels the canonicalized LabelSet used to associate with this metric handle.
    */
-  getHandle(labelValues: string[]): T {
-    const hash = hashLabelValues(labelValues);
+  getHandle(labels: types.LabelSet): T {
+    const hash = hashLabelValues(Object.values(labels));
```

Comment:
Consider storing the hash value of a label set values in the actual object. This way, you don't have to perform an expensive conversion every single time you get a handle.

Reply:
This leads to the question: should LabelSet objects be canocalized by default, or only when created by `meter.Labels(...)`?
Thoughts? @mayurkale22 @xiao-lix

Reply:
It was my impression that storing the canonicalized/serialized version in the LabelSet object in order to avoid re-computation was one of the primary reasons for their existence.